homeassistant/components/rflink/sensor.py

```diff
@@ -61,25 +64,109 @@
         state_class=SensorStateClass.MEASUREMENT,
     ),
     SensorEntityDescription(
-        key="average_windspeed",
-        name="Average windspeed",
-        device_class=SensorDeviceClass.WIND_SPEED,
+        key="battery",
+        name="Battery",
+        icon="mdi:battery",
```

Comment:
I tried that but in the frontend, device_class=BATTERY means that the battery sensor has a value between 0 and 100%. With my sensors I have `ok` or `low` instead of a number. So the icon is `mdi:battery-unknown`

Reply:
Ummmm, you are right...
No `device_class` for RFLink batteries 👍🏻

Reply:
If a sensor only has two possible values it should probably be a binary sensor instead.